common/src/androidTest/java/com/microsoft/identity/common/CommandDispatcherTest.java

```diff
@@ -365,6 +369,48 @@ public void run() {
         }
     }
 
+    @Test
+    public void testCanRecoverFromHangingTask() throws Exception {
+        final AtomicInteger integer = new AtomicInteger(0);
+        final CountDownLatch latch = new CountDownLatch(1);
+        Context context = InstrumentationRegistry.getInstrumentation().getContext();
+        CommandCallback callback = new CommandCallback() {
+
+            @Override
+            public void onTaskCompleted(Object o) {
+
+            }
+
+            @Override
+            public void onError(Object error) {
+
+            }
+
+            @Override
+            public void onCancel() {
+
+            }
+        };
+
+        CommandDispatcher.beginInteractive(new InteractiveTokenCommand(BrokerInteractiveTokenCommandParameters.builder().androidApplicationContext(context).build(), (BaseController) null, callback, null) {
```

Comment:
This line is too long and very difficult to read here on GitHub, may be split it across multiple lines

Reply:
We should document somewhere what our threshold is... if not for ourselves, for external contributors. I usually say 100 chars. AOSP says the same, I think we should set that in 'stone' somewhere

https://source.android.com/setup/contribute/code-style#limit-line-length

Adam and I chatted on briefly a week or two ago and, while most (all?) of us have widescreen monitors, I usually do a lot of v-splitting in my editor and having nice narrow columns usually prevents me having the horizontally scroll. I am less insistent on this _for tests_, though we might want to codify this somewhere

As a random example (just opening some files), the left-pane goes off-screen just a bit in the below screenshot, hence my advocacy for approx. 100 chars
![image](https://user-images.githubusercontent.com/990063/111409687-ad8e5480-8694-11eb-9d6f-437f1ee92cd5.png)